content/blog/sprint-review-2317.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Wazo Platform 23.17 Released
+date: 2023-12-19T08:30:00
+author: The Wazo Authors
+category: Wazo Platform
+tags: wazo-platform, development
+slug: release-review-2317
+status: published
+---
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 23.17 release.
+
+## New Features in This Release
+- **Agents**: Agents number are now tenant aware.  Multiple tenants may now use the same number for agents without conflict.
+
+## Bug Fixes
+- **CDR**: 'user_uuid' filter on /cdr API now only matches source and final destination participants
+- **call-logd**: fixed critical bug in `wazo-call-logd-sync-db` causing loss of data
```

Comment:
The fix has been backported at the beginning of the sprint. I would not mention it here
```suggestion
```